admin-dev/themes/default/template/controllers/modules_positions/list_modules.tpl

```diff
@@ -65,16 +65,12 @@
 						</div>
 					</div>
 				</div>
-				<div class="row">
-					<div class="col-sm-12">
-							<p class="checkbox">
-								<label class="control-label" for="hook_position">
-									<input type="checkbox" id="hook_position"/>
-									{l s='Display non-positionable hooks' d='Admin.Design.Feature'}
-								</label>
-							</p>
-					</div>
-				</div>
+				<p class="checkbox">
+					<label class="control-label" for="hook_position">
+						<input type="checkbox" id="hook_position"/>
+						{l s='Display non-positionable hooks' d='Admin.Design.Feature'}
+					</label>
+				</p>
```

Comment:
Are you sure this content will behave as expected in the grid outside of any row or form-group ?

Reply:
Yes dont worry